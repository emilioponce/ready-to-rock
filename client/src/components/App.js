import React from 'react'
import Header from './Header'
import Content from './Content'
import Footer from './Footer'

import styles from './App.module.css'

const App = () => (
  <div className={styles.app}>
    <Header />
    <Content />
    <Footer />
  </div>
)

export default App
