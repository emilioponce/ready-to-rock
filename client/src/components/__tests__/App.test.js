import React from 'react'
import ReactDOM from 'react-dom'
import App from '../App'
import { shallow } from 'enzyme'

const component = <App />

describe('Component App', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(component, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('snapshot renders correctly', () => {
    const wrapper = shallow(component)
    expect(wrapper).toMatchSnapshot()
  })
})
