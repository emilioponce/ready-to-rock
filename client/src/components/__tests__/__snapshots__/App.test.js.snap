// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component App snapshot renders correctly 1`] = `
<div
  className="app"
>
  <Header />
  <Content />
  <Footer />
</div>
`;
